### ISSUE TITLE
Washington State Payroll Taxes for 2023

### ISSUE THREAD

**Issue by DTrim99** [PolicyEngine/policyengine-us#4936]
Washington's Family and Medical Leave Payroll Tax is 0.8% up to the Social Security cap ($160,200). Employers cover 0.22%, and employees cover 0.58%. Employers with less than 50 employees are exempt. 

Washington Care's payroll tax is 0.58% (uncapped, withheld from employees).

[Reference](https://www.paynw.com/blog/washington-state-payroll-taxes)

**Comment by MaxGhenis** [comment]
Agreed @PavelMakarchuk. We don't currently model any state-level payroll taxes. I can't immediately find whether these count for the SALT deduction.

**Comment by DTrim99** [comment]
For 2024, the payroll tax rate is 0.74% up to the 2024 Social Security cap ($168,600). Employers pay 0.21%; employees pay 0.53%. 

[Reference](https://paidleave.wa.gov/estimate-your-paid-leave-payments/)

**Comment by DTrim99** [comment]
Closing as implemented. WA PFML and WA Cares payroll contributions are now modeled via `wa_employee_state_payroll_tax` and `wa_employer_additional_state_payroll_tax`, which aggregate `wa_employee_paid_leave_contribution`, `wa_employee_long_term_care_contribution`, and `wa_employer_paid_leave_contribution`. Rates are parameterized under `parameters/gov/states/wa/tax/payroll/{paid_leave,long_term_care}/` and reflect current RCW 50A.10.030 and 50B.04.080.
